### PR TITLE
FHIR-35776 - Cut and paste error in diagram

### PR DESCRIPTION
**Pull request by awatson1978**
Preview
https://build.fhir.org/ig/HL7/davinci-deqm/branches/FHIR-35776/

### CODE REVIEW

**Comment by awatson1978** [comment]
New image:
COL Worklow Diagram - Data Exchange Indiv collect-data

**Comment by p9g** [comment]
Should this be to merge into 2023-Jan-Ballot branch?  Master is currently for the 3.1.0 update.

**Comment by p9g** [comment]
The vertical grey bars are not aligned on the left

**Comment by awatson1978** [comment]
Revised workflow diagram:

COL Worklow Diagram - Data Exchange Indiv collect-data

**Comment by p9g** [comment]
Colors should match other diagrams on [that page](https://hl7.org/fhir/us/davinci-deqm/STU3/col.html#summary-measurereport)
Text should be contained within box boundaries.

**Comment by awatson1978** [comment]
Not sure why the source diagrams have different colors.  Here is the updated diagram.

COL Worklow Diagram - Data Exchange Indiv collect-data

**Comment by awatson1978** [comment]
Looks good:
https://build.fhir.org/ig/HL7/davinci-deqm/branches/FHIR-35776/col.html

**Review by brynrhodes — DISMISSED** [review]
Why does a fix to this diagram add a profile definition from QM IG?

File: FHIR-us-davinci-deqm.xml [hunk 1]
```diff
@@ -13,6 +13,7 @@
 <artifactPageExtension value="-definitions"/>
 <artifactPageExtension value="-examples"/>
 <artifactPageExtension value="-mappings"/>
+<artifact id="StructureDefinition/module-definition-library-cqfm" key="StructureDefinition-module-definition-library-cqfm" name="CQFM Module Definition Library"/>
```
Comment (p9g) [inline FHIR-us-davinci-deqm.xml:16]: Why was this added?

File: input/examples/StructureDefinition-module-definition-library-cqfm.json [hunk 2]
```diff
@@ -0,0 +1,222 @@
+{
```
Comment (p9g) [inline input/examples/StructureDefinition-module-definition-library-cqfm.json:1]: Why is this file added?

File: FHIR-us-davinci-deqm.xml (hunk 1 above)
Comment (awatson1978) [reply FHIR-us-davinci-deqm.xml]: I got no clue.  Wasn't anything I intentionally added.  But I do have a tendency to rely on the `_genonce.sh` tool to re-build the IG, and I'm fairly sure that I ran `_updatePublisher.sh` sometime in the past month.  Maybe an update to dependencies caused it to get compiled or fetched from elsewhere?

File: input/examples/StructureDefinition-module-definition-library-cqfm.json (hunk 2 above)
Comment (awatson1978) [reply input/examples/StructureDefinition-module-definition-library-cqfm.json]: I have no clue.  It wasn't anything I did intentionally.  I have no clue.  Wasn't anything I intentionally added.  But I do have a tendency to rely on the `_genonce.sh` tool to re-build the IG, and I'm fairly sure that I ran `_updatePublisher.sh` sometime in the past month.  Maybe an update to dependencies caused it to get compiled or fetched from elsewhere?  I think this entry was auto-added.

File: FHIR-us-davinci-deqm.xml (hunk 1 above)
Comment (awatson1978) [reply FHIR-us-davinci-deqm.xml]: File removed.

**Comment by p9g** [comment]
Thanks for removing the QMIG stuff.
The jpg diagram for Figure 3-10 still has one box with text outside the bounds of the rectangle.
Some of the .sh scripts were updated more recently in master branch, I'm not sure about any conflicts, so please assess those files.
(The Files changed tab says "empty file" for 2 of the files, but I don't know if that means your branch deleted the file, or just removed any changes.)

**Comment by awatson1978** [comment]
JPG updated.  I have no idea why it says 'Empty file'.... the file exists in the branch.  Also pulled and merged the master branch, and it compiles correctly.

**Comment by awatson1978** [comment]
@brynrhodes @p9g

**Review by p9g — APPROVED** [review]
Diagram looks good.

**Review by brynrhodes — APPROVED** [review]
Other than resolving the conflicts, looks good

**Comment by yanheras** [comment]
Close this pull request. 
This is replaced by a new pull request (https://github.com/HL7/davinci-deqm/pull/66). The update to the figure 3-10 diagram is made directly using the original source image.
